src/components/BorderedList/BorderedList.tsx

```diff
@@ -0,0 +1,30 @@
+import type { ListProps } from '@digdir/design-system-react';
+import { List } from '@digdir/design-system-react';
+import cn from 'classnames';
+import * as React from 'react';
+
+import classes from './BorderedList.module.css';
+
+export type BorderedListProps = {
+  /**
+   * Style of the border separating the items
+   */
+  borderStyle?: 'solid' | 'dashed';
+} & ListProps;
+
+export const BorderedList = ({
+  borderStyle = 'dashed',
+  children,
+  className,
+  ...rest
+}: BorderedListProps) => {
+  return (
+    <List
+      className={cn(classes.borderedList, { [classes[borderStyle]]: borderStyle }, className)}
+      style={{ paddingLeft: 0 }}
```

Comment:
I don't see why we should have classes and set style prop if we can do it all with one of those. Can't you set these styles in css classes as well?

Reply:
I briefly touched upon this in the other thread, but the reason here is that we're battling the internal CSS of List, which adds a left padding we want to erase.

The only way to do this (that I know of) is by overpowering it in terms of priority, meaning either inline styling or with an !important tag, and while I've argued for the use of !important in the other thread, I don't think we should use it here as it is not the proverbial "end of the line". This is a reusable component, and it isn't unreasonable to assume there might be cases of conflicting css down the line.